statina/API/v2/endpoints/sample.py

```diff
@@ -0,0 +1,261 @@
+import io
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Literal, Optional
+
+from fastapi import APIRouter, Depends, Form, Query, Security
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import FileResponse, JSONResponse
+
+from statina.adapter import StatinaAdapter
+from statina.API.v2.endpoints.user import get_current_active_user
+from statina.config import get_nipt_adapter
+from statina.crud import update
+from statina.crud.find import find
+from statina.crud.find.plots.zscore_plot_data import (
+    get_abn_for_samp_tris_plot,
+    get_normal_for_samp_tris_plot,
+    get_sample_for_samp_tris_plot,
+)
+from statina.models.database import Batch, DataBaseSample, User
+from statina.models.server.plots.ncv import Zscore131821, ZscoreSamples
+from statina.models.server.sample import Sample
+from statina.parse.batch import validate_file_path
+
+router = APIRouter(prefix="/v2")
+
+
+@router.get("/samples/", response_model=List[Sample])
+def samples(
+    page_size: Optional[int] = Query(5),
+    page_num: Optional[int] = Query(0),
+    current_user: User = Security(get_current_active_user, scopes=["R"]),
+    adapter: StatinaAdapter = Depends(get_nipt_adapter),
+):
+    """Get samples"""
+    samples: List[DataBaseSample] = find.samples(
+        adapter=adapter, page_size=page_size, page_num=page_num
+    )
+    return JSONResponse(content=jsonable_encoder(samples))
+
+
+@router.get("/sample/{sample_id}", response_model=Sample)
+def sample(
+    sample_id: str,
+    current_user: User = Security(get_current_active_user, scopes=["R"]),
+    adapter: StatinaAdapter = Depends(get_nipt_adapter),
+):
+    """Get sample with id"""
+
+    sample: DataBaseSample = find.sample(sample_id=sample_id, adapter=adapter)
+
```

Comment:
Aslo this endpoint need to return chrom_abnorm, status_classes and maybe more. We can have a look. Should be a responce model anyway if so.

Reply:
Or like this...: 
**The info box section** will need Sample and Batch object
**The table section** will need Sample and the chrom_abnorm list ([14, 18, 21,X0,...]) (could be hardcoded in frontend if we want..)
**The plot section** will need normal_data abnormal_data and sample_data

So maybe create another endpoint for info-box section? What you think?